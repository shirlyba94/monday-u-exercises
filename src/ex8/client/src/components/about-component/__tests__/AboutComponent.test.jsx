import renderer from "react-test-renderer";
import AboutComponent from "../AboutComponent";

it("renders correctly", () => {
    const tree = renderer.create(
        <AboutComponent />
    ).toJSON();
   expect(tree).toMatchSnapshot();
});