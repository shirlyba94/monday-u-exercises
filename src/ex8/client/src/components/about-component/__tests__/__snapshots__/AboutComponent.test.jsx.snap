// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1
  aria-label=""
  className="heading-component element-type-h1 size-large single-line-ellipsis"
  id=""
>
  About me page
</h1>
`;
